blocktopmenu.php

```diff
@@ -52,7 +52,7 @@ public function __construct()
     {
         $this->name = 'blocktopmenu';
         $this->tab = 'front_office_features';
-        $this->version = '2.2.2';
+        $this->version = '2.3';
```

Comment:
Hi,

Thank you for your PR. No need to uupdate version number in your PR, thank you.

Regards